adapter_nina.go

```diff
@@ -0,0 +1,113 @@
+//go:build nina || nano_rp2040
+
+package bluetooth
+
+import (
+	"machine"
+
+	"time"
+)
+
+// Adapter is a dummy adapter: it represents the connection to the NINA fw.
+type Adapter struct {
```

Comment:
Doesn't sound very dummy to me? It's an adapter, even if the actual BLE stack is elsewhere.

Reply:
Changed.